core/src/epicli/data/common/ansible/playbooks/roles/vault/templates/vault/policy-admin.hcl.j2

```diff
@@ -0,0 +1,50 @@
+# {{ ansible_managed }}
+# Auth methods
+path "auth/*"
+{
+  capabilities = ["create", "read", "update", "delete", "list", "sudo"]
+}
+
+path "sys/auth/*"
+{
+  capabilities = ["create", "update", "delete", "sudo"]
+}
+
+path "sys/auth"
+{
+  capabilities = ["read"]
+}
+
+# Policies
+path "sys/policies/acl"
+{
+  capabilities = ["list"]
+}
+
+path "sys/policies/acl/*"
+{
+  capabilities = ["create", "read", "update", "delete", "list", "sudo"]
+}
+
+# Secrets engines
+path "sys/mounts/*"
+{
+  capabilities = ["create", "read", "update", "delete", "list", "sudo"]
+}
+
+path "sys/mounts"
+{
+  capabilities = ["read"]
+}
+
+# Health checks
+path "sys/health"
+{
+  capabilities = ["read", "sudo"]
+}
+
+# KV secrets from path {{ specification.secret_mount_path }}
+path "{{ specification.secret_mount_path }}/*"
```

Comment:
why is that configurable?

Reply:
Because user can have his own path to secrets e.g. he have created something in test environment at different path and he also wants to use it in the production.